terra-jupyter-aou/Dockerfile

```diff
@@ -30,10 +30,16 @@ ENV PIP_USER=false
 ENV PYTHONPATH $PYTHONPATH:/usr/lib/spark/python
 
 RUN pip3 install \
+      igv-jupyter \
       nbstripout \
       papermill \
       "git+https://github.com/all-of-us/workbench-snippets.git#egg=terra_widgets&subdirectory=py"
 
+RUN jupyter nbextension enable --py igv
```

Comment:
I realize there is some capability to enable this via the Leo API. However, since I had to install the package here anyways, I didn't see much value in splitting the configuration up. Let me know if I'm missing something here.

Reply:
We can probably move this to the python image too

Reply:
I don't know whether it's useful in general Terra since you have a built-in IGV already - I'll probably leave it here for now